db/db.go

```diff
@@ -737,67 +743,6 @@ func SaveBlock(block *types.Block, forceSlotUpdate bool) error {
 	return nil
 }
 
-func UpdateChainHead(head *types.ChainHead) error {
```

Comment:
The Postgres table `chain_head` is not used anymore and should be removed with a migration file.